Riot/Modules/Authentication/AuthenticationViewController.m

```diff
@@ -1600,14 +1605,13 @@ - (void)showCustomHomeserver:(NSString*)homeserver andIdentityServer:(NSString*)
 
 - (void)keyVerificationCoordinatorBridgePresenterDelegateDidComplete:(KeyVerificationCoordinatorBridgePresenter * _Nonnull)coordinatorBridgePresenter otherUserId:(NSString * _Nonnull)otherUserId otherDeviceId:(NSString * _Nonnull)otherDeviceId
 {
+    // Set outgoing key requests back
+    [coordinatorBridgePresenter.session.crypto setOutgoingKeyRequestsEnabled:YES onComplete:nil];
```

Comment:
We must not force the reactivation of it because of what is explained at https://github.com/vector-im/element-ios/pull/3972/files#diff-65a750f2afa90226c3484b2e47732b3f945977214e305e3883edf4015e4e3ea3R1388-R1389.

Enabling it will launch share requests in parallel of key backup restore.